src/connection.cpp

```diff
@@ -766,6 +782,19 @@ namespace oxen::quic
         // config values
         params.initial_max_streams_bidi = (uconfig.max_streams) ? uconfig.max_streams : DEFAULT_MAX_BIDI_STREAMS;
 
+        if (uconfig.datagram_support)
+        {
+            params.max_datagram_frame_size = 65535;
+            // default ngtcp2 values set by ngtcp2_settings_default_versioned
```

Comment:
The only one i saw that `ngtcp2_settings_default` assigns is 

`settings->max_tx_udp_payload_size = 1500 - 48;`

I set it again explicitly in case we find the given overhead they estimated was insufficient, but I can absolutely remove it if its not helpful

The other one that `ngtcp2_transport_params_default` sets 

`params->max_udp_payload_size = NGTCP2_DEFAULT_MAX_RECV_UDP_PAYLOAD_SIZE;`

I set it to 65535 as per the rfc; their macro is 65527